.gitignore

```diff
@@ -24,42 +27,19 @@ Servers/
 .java-version
 
 # Maven
+target/
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+pom.xml.next
+release.properties
+dependency-reduced-pom.xml
+buildNumber.properties
+.mvn/timing.properties
 .mvn/wrapper/maven-wrapper.jar
-core/target
-plugins/convention/target
-plugins/embeddedjsp/target
-plugins/osgi/target
-plugins/cdi/target
-plugins/spring/target
-plugins/jfreechart/target
-apps/rest-showcase/target
-apps/mailreader/target
-apps/portlet/target
-apps/blank/target
-apps/jboss-blank/target
-apps/mailreader/target
-apps/portlet/target
-apps/showcase/target
-apps/target
-bundles/admin/target
-plugins/config-browser/target
-plugins/jasperreports/target
-plugins/json/target
-plugins/portlet-tiles/target
-plugins/portlet/target
-plugins/sitemesh/target
-plugins/javatemplates/target
-bundles/demo/target
-plugins/rest/target
-plugins/plexus/target
-plugins/testng/target
-plugins/dwr/target
-plugins/gxp/target
-plugins/oval/target
-plugins/junit/target
-plugins/tiles/target
-bundles/target
-plugins/target
-target
+
 plugins/testng/test-output
 test-output
+
+# Sonar
+/.sonar/
```

Comment:
Do we need that leading slash?

Reply:
> Do we need that leading slash?

Not necessarily, but according to https://git-scm.com/docs/gitignore "A leading slash matches the beginning of the pathname. For example, "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c".

Because there is only one `.sonar` dir in the root IMHO with the leading slash this is more specific.

Reply:
👍